### ISSUE TITLE
inconsistency in commands

### ISSUE THREAD

**Issue by surajnarwade** [redhat-developer/odo#391]
if we want to delete app, url or component, we have commands like,

```
odo app delete <appname>

odo url delete <name>

odo delete <componentname>
```

But if you see storage part, it's

```
odo storage remove <name>
```

for consistency in UX, I think we should do,

```
odo storage delete <name>
```

**Comment by syamgk** [comment]
I like to work on this

**Comment by surajnarwade** [comment]
@containscafeine  @kadel @cdrage WDYT ?

**Comment by cdrage** [comment]
Seems good to me (better consistency)

**Comment by kadel** [comment]
closed in #407
